tests/rados/test_pg_autoscale_flag.py

```diff
@@ -51,13 +51,9 @@ def run(ceph_cluster, **kw):
 
     # Getting the autoscale configurations after setting the flag
     # all the pools should have autoscale set to off
-    cmd = "ceph osd pool autoscale-status"
-    pool_status = rados_obj.run_ceph_command(cmd=cmd)
-
-    for entry in pool_status:
-        if entry["pg_autoscale_mode"] == "on":
-            log.error(f"Pg autoscaler not turned off for pool : {entry['pool_name']}")
-            return 1
```

Comment:
Line 45-53 can also be removed as it is now part of `set_autoscale_flag` common module

Reply:
done. removed all invalid lines